lms/envs/common.py

```diff
@@ -1546,7 +1546,7 @@ def _make_locale_paths(settings):  # pylint: disable=missing-docstring
     'UGLIFYJS_BINARY': 'node_modules/.bin/uglifyjs',
 }
 
-STATICFILES_STORAGE = 'openedx.core.storage.ProductionStorage'
+STATICFILES_STORAGE = os.environ.get('STATICFILES_STORAGE', 'openedx.core.storage.ProductionStorage')
```

Comment:
Why use os.environ instead of using the settings file where all the other settings live?

Reply:
Because I need to be able to choose at run-time whether to write to s3 or to disk. (In particular, I'm planning to run the asset builder twice, once for local and the other for s3, during the transition period). I'll add a comment to that effect.

Reply:
Cool makes sense, since editing the config in between the runs will be much harder.

Reply:
Updated.